### PR TITLE
#1442 Enhanced order search

### PR DESCRIPTION
**Pull request by mikethetike**
### References Issues:
Connected to #1442 

### Description:
Add extra search fields for admin/orders

### Migrations
 * No Migrations

### Environment Variables
 * No change

### CODE REVIEW

File: db/src/models/orders.rs [hunk 1]
```diff
@@ -721,13 +721,17 @@ impl Order {
...
         email: Option<&str>,
+        phone: Option<&str>,
         name: Option<&str>,
         ticket_type_id: Option<Uuid>,
         partial_promo_code: Option<&str>,
         box_office_sales: bool,
         online_sales: bool,
         web_sales: bool,
         app_sales: bool,
+        _payment_method: Option<PaymentMethods>,
```
Comment (Timo614) [inline db/src/models/orders.rs]: Is this leftover from an earlier iteration? Or out of scope and handling in a future branch?

File: db/src/models/orders.rs [hunk 2]
```diff
@@ -813,6 +829,20 @@ impl Order {
...
+            bind_no = bind_no + 1;
+            query = query
+                .sql(format!(" and o.order_date >= ${} ", bind_no))
+                .bind::<sql_types::Timestamp, _>(date);
+        }
+
+        if let Some(date) = end_date {
+            bind_no = bind_no + 1;
+            query = query
+                .sql(format!(" and o.order_date >= ${} ", bind_no))
```
Comment (Timo614) [inline db/src/models/orders.rs]: I think this filter has a typo and order_date should be `<=`

Comment (mikethetike) [reply db/src/models/orders.rs]: Good catch

File: db/src/models/orders.rs (hunk 1 above)
Comment (mikethetike) [reply db/src/models/orders.rs]: I was waiting on a response on the issue. I think we already have the fields we require

**Comment by mikethetike** [comment]
I also reduced the 401's to log as Info instead of Warn. That could have been in a seperate PR, but this one will probably go in soon

**Pull request merged**